src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -1524,18 +1524,50 @@ enum MultiModalMode {
   parent
 }
 
+"OccupancyStatus to be exposed in the API. The values are based on GTFS-RT"
 enum OccupancyStatus {
-  "The vehicle or carriage has a few seats available."
+  """
+  The vehicle or carriage can currently accommodate only standing passengers and has limited
+  space for them. There isn't a big difference between this and `full` so it's possible to
+  handle them as the same value, if one wants to limit the number of different values.
+  SIRI nordic profile: merge into `standingRoomOnly`.
+  """
+  crushedStandingRoomOnly
+  """
+  The vehicle is considered empty by most measures, and has few or no passengers onboard, but is
+  still accepting passengers. There isn't a big difference between this and `manySeatsAvailable`
+  so it's possible to handle them as the same value, if one wants to limit the number of different
+  values.
+  SIRI nordic profile: merge these into `manySeatsAvailable`.
+  """
+  empty
+  """
+  The vehicle or carriage has a few seats available.
+  SIRI nordic profile: less than ~50% of seats available.
+  """
   fewSeatsAvailable
-  "The vehicle or carriage is considered full by most measures, but may still be allowing passengers to board."
+  """
+  The vehicle or carriage is considered full by most measures, but may still be allowing
+  passengers to board.
+  """
```

Comment:
```suggestion
  SIRI nordic profile: merge these into `standingRoomOnly`.
  """
```

Reply:
I think this value exists in the nordic profile as well. I didn't include the documentation here because it was so similar to the gtfs rt documentation "full (close to or at full capacity)" https://enturas.atlassian.net/wiki/spaces/PUBLIC/pages/637370392/SIRI-ET#SIRI-ET-VehicleOccupancyStructure . I can include if you want but I don't think we should suggest merging into standingRoomOnly?

Reply:
Ok, then it is fine.